### DISCUSSION TITLE
Is there a way to live preview a pdf using a specified ratio

### DISCUSSION THREAD

**Discussion by lisiur** [sxyazi/yazi#1133] [1. Q&A] [answered]
I love yazi! It has an impressive pdf live preview function. Currently I can preview a pdf using this command:
```
yazi ./demo.pdf
```
But it uses the default ratio(1:4:3) layout. How can i preview it using a specified layout like 0:0:1.

**Comment by sxyazi** [comment, marked as answer]
Here's a tip to maximize the preview area, https://yazi-rs.github.io/docs/tips#max-preview, which I think could be helpful. 

You can also refer to its code to make it smarter, such as automatically maximizing when the hovered file is a PDF, without needing a single key press.
